test/GlobalVarNoAddrspace.ll

```diff
@@ -0,0 +1,16 @@
+; RUN: llvm-as %s -o %t.bc
+; RUN: not llvm-spirv %t.bc 2>&1 \
+; RUN: | FileCheck %s --check-prefix=CHECK-ERROR
+
+; CHECK-ERROR: InvalidInstruction: Can't translate llvm instruction:
+; CHECK-ERROR-NEXT: Global variable can not have Function storage class. Consider setting a proper address space.
```

Comment:
Would it make sense to move this test into `test/negative/`?

Reply:
Thank you @svenvh We have started to disperse negative tests over the whole test suite, and tend to group it by subject rather than by the negative criteria.

Reply:
This test doesn't appear to be grouped by subject as it's directly under `test/` (ideally we should aim to reduce the number of tests in the top-level directory and group them by subject as you say).

Reply:
You are absolutely right, I've been thinking about this in the same terms when creating the PR. If it's a blocker, I will create another commit, grouping two tests GlobalVarNoAddrspace and GlobalVarAddrspace together in a new folder. However, it looked for me like a bit exaggeration to create a new folder for just two new tests.

You are right though that the top-level directory should be reordered by topics. May we plan this as a wider scope activity, and address the top-level directory next time, or you would prefer me to rework this PR and place those two tests into a new dedicated folder?

Reply:
I agree that a directory with just 2 tests is a bit of overkill.